.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+/.github/ @kroxylicious/admins
```

Comment:
Intent of this line is to follow Github [best practice](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#:~:text=To%20protect%20a%20repository%20fully%20against%20unauthorized%20changes%2C%20you%20also%20need%20to%20define%20an%20owner%20for%20the%20CODEOWNERS%20file%20itself.).